Formula/solarus.rb

```diff
@@ -17,7 +18,7 @@ class Solarus < Formula
   depends_on "libmodplug"
   depends_on "libogg"
   depends_on "libvorbis"
-  depends_on "luajit"
+  depends_on "luajit-openresty"
```

Comment:
Officially, upstream lists Lua 5.1 / LuaJIT 2.0 as dependency, so could be safer to only enable `luajit-openresty` on ARM (https://gitlab.com/solarus-games/solarus/-/blob/dev/compilation.md#external-dependencies)